src/providers/tpm/utils.rs

```diff
@@ -93,27 +95,64 @@ pub fn to_response_status(error: Error) -> ResponseStatus {
 // The PasswordContext is what is stored by the Key Info Manager.
 #[derive(Serialize, Deserialize, Zeroize)]
 pub struct PasswordContext {
+    /// This value is kept for legacy purposes, to aid in the migration process
+    pub context: TpmsContext,
+    /// This value is confidential and needs to be zeroized by its new owner.
+    pub auth_value: Vec<u8>,
+    pub key_material: KeyMaterial,
+}
```

Comment:
Could this struct be encapsulated, removing the use of `pub` properties?

Reply:
Done